Resources/Textures/Objects/Consumable/Drinks/the_martinez.rsi/meta.json

```
@@ -0,0 +1 @@
+{"version": 1, "size": {"x": 32, "y": 32}, "license": "CC-BY-SA-3.0", "copyright": "https://github.com/discordia-space/CEV-Eris/raw/f7aa28fd4b4d0386c3393d829681ebca526f1d2d/icons/obj/drinks.dmi", "states": [{"name": "icon"}]}
```

```suggestion
{"version": 1, "size": {"x": 32, "y": 32}, "license": "CC-BY-SA-3.0", "copyright": "https://github.com/discordia-space/CEV-Eris/commit/93dd3fe8aa166a204ae90c520c273e129b574a21", "states": [{"name": "icon"}]}
```

The copyright should link to a commit for the drink sprite and not the dmi file directly.  I changed this for you.